api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -115,6 +115,7 @@
 
 	private static final String ORDER_GROUP_ATTRIBUTES = "org/openmrs/api/include/OrderServiceTest-createOrderGroupAttributes.xml";
 
+	private static final String ORDER_TYPES = "org/openmrs/api/include/OrderServiceTest-getOrderTypes.xml";
```

Comment:
Would be great to put a new line between.

Reply:
Oops sorry , i removed this in the previous editings.